projects/plugins/jetpack/views/admin/network-settings.php

```diff
@@ -1,86 +1,67 @@
-			</tr>
-/**/ ?>
-			<tr valign="top">
-				<th scope="row"><label for="sub-site-override"><?php _e( 'Sub-site override', 'jetpack' ); ?></label></th>
-				<td>
-					<input type="checkbox" name="sub-site-connection-override" id="sub-site-override" value="1" <?php checked($options['sub-site-connection-override']); ?> />
-					<label for="sub-site-override"><?php _e( 'Allow individual site administrators to manage their own connections (connect and disconnect) to <a href="//wordpress.com">WordPress.com</a>', 'jetpack' ); ?></label>
+					<input type="checkbox" name="sub-site-connection-override" id="sub-site-override" value="1" <?php checked( $data['options']['sub-site-connection-override'] ); // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable ?> />
+					<label for="sub-site-override"><?php esc_html_e( 'Allow individual site administrators to manage their own connections (connect and disconnect) to WordPress.com', 'jetpack' ); ?></label>
 				</td>
 			</tr>
 
 			<tr valign="top">
-				<th scope="row"><label for="sub-site-override"><?php _e( 'Protect whitelist', 'jetpack' ); ?></label></th>
-				<td>
-					<p><strong><?php printf( __( 'Your current IP: %s', 'jetpack' ), jetpack_protect_get_ip() ); ?></strong></p>
-					<textarea name="global-whitelist" style="width: 100%;" rows="8"><?php echo implode( PHP_EOL, $jetpack_protect_whitelist['global'] ); ?></textarea> <br />
-					<label for="global-whitelist"><?php _e('IPv4 and IPv6 are acceptable. Enter multiple IPs on separate lines. <br />To specify a range, enter the low value and high value separated by a dash. Example: 12.12.12.1-12.12.12.100', 'jetpack' ); ?></label>
-				</td>
-			</tr>
-<?php /* Remove the toggles for 2.9, re-evaluate how they're done and added for a 3.0 release. They don't feel quite right yet.
-			<tr>
-				<th scope="row"><label for="manage_auto_activated_modules">Manage modules</label></th>
+				<th scope="row"><label for="sub-site-override"><?php esc_html_e( 'Protect whitelist', 'jetpack' ); ?></label></th>
 				<td>
-					<input type="checkbox" name="manage_auto_activated_modules" id="manage_auto_activated_modules" onclick="jQuery('#jpms_settings_modules').toggle();" value="1" <?php checked( $options['manage_auto_activated_modules'] ); ?>/>
-					<label for="manage_auto_activated_modules">Control which modules are auto-activated</label>
+					<p><strong>
+					<?php
+					$protect_ip = jetpack_protect_get_ip();
+					if ( ! empty( $protect_ip ) ) {
+						printf(
+							/* Translators: placeholder is an IP address. */
+							esc_attr__( 'Your current IP: %1$s', 'jetpack' ),
+							esc_attr( $protect_ip )
+						);
+					}
+					?>
+					</strong></p>
+					<?php
+					echo '<textarea name="global-whitelist" style="width: 100%;" rows="8">';
+
+					foreach ( $data['jetpack_protect_whitelist']['global'] as $ip ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable
+						echo esc_attr( $ip ) . "\n";
```

Comment:
`esc_textarea`?

Reply:
Since we're only dealing with IPs (or IP ranges) here and not the full textarea, I think it is fine to keep escaping like we do above. I updated it together with the one above, in 324ac72bf0f0027bd1502f58d5d970cab52c5e87.